services/ui-src/src/components/report/Elements.tsx

```diff
@@ -88,6 +89,39 @@ export const subHeaderElement = (props: PageElementProps) => {
   );
 };
 
+export const subHeaderMeasureElement = (_props: PageElementProps) => {
+  const { report, currentPageId } = useStore();
+
+  //search for the measure that the currentId is tied to, it maybe a child so we have to check the dependents too.
+  const parentMeasure = report?.measureLookup.defaultMeasures.find(
```

Comment:
if the report pruning story gets in first, this is going to need to adjust to look at the parent page .dependtPages.

If not, the other story will need to adjust

Reply:
```suggestion
  const measure = report?.pages.find(
    (page) =>
      isMeasurePageTemplate(page) &&
      page.dependentPages &&
      page.dependentPages.find((child) => child.template === currentPageId)
  ) as MeasurePageTemplate;
  
  ```